subsys/bluetooth/audio/aics.c

```diff
@@ -0,0 +1,826 @@
+/*  Bluetooth AICS
+ *
+ * Copyright (c) 2020 Bose Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * AICS is a secondary service that can be included by other services.
```

Comment:
The license should probably be separated from the description.

Shouldn't Nordic be added? And 2021

Reply:
Do we have any guidelines on dealing with this? 

There weren't any similar comments for my VOCS PR (https://github.com/zephyrproject-rtos/zephyr/pull/31893). Regarding the year; should it be the year that Nordic started working on it, a 2020-2021 or the year Nordic upstreamed this file? I've heard different expectations.

Reply:
But added Nordic to the files.

Reply:
I believe Nordic policy is the year the work was started